chart/templates/proxy-configmap.yaml

```diff
@@ -51,54 +51,15 @@ data:
       }
     }
 {{- end }}
-{{- if .Values.vsxRegistry.proxy }}
+{{- if not .Values.components.openVsxProxy.disabled }}
```

Comment:
I am not sure why do we need this? I thought the load balancer should be configured to hit Open VSX Proxy directly.

Reply:
It's mainly for self-hosted. And for core-dev.

Reply:
ok, thanks for explaining 👍